lib/media/media_source_engine.js

```diff
@@ -374,6 +378,16 @@ shaka.media.MediaSourceEngine = class {
     }
   }
 
+  /**
+   * Called by the Player to provide an updated configuration any time it
+   * changes. Must be called at least once before init().
+   *
+   * @param {shaka.extern.MediaSourceConfiguration} config
+   */
+  configure(config) {
+    this.config_ = config;
+  }
+
```

Comment:
Will this value ever change at runtime? If not, could it be sent in via the constructor to eliminate this extra step?